audbackend/core/backend/base.py

```diff
@@ -261,14 +268,21 @@ def create(
         Args:
             host: host address
             repository: repository name
+            auth: access token
+                for possible authentication,
+                e.g. username, password tuple
```

Comment:
"access token" already sounds quite specific to me. Maybe we keep it more general: 

```suggestion
            auth: object used for authentication,
                e.g. a tuple with username and password
```